tests/generate_code.sh

```diff
@@ -15,15 +15,17 @@
 # limitations under the License.
 set -e
 
-../flatc --cpp --java --kotlin  --csharp --dart --go --binary --lobster --lua --python --js --ts --php --rust --grpc --gen-mutable --reflect-names --gen-object-api --gen-compare --no-includes --cpp-ptr-type flatbuffers::unique_ptr  --no-fb-import -I include_test monster_test.fbs monsterdata_test.json
+../flatc --cpp --java --kotlin  --csharp --dart --go --binary --lobster --lua --js --ts --php --rust --grpc --gen-mutable --reflect-names --gen-object-api --gen-compare --no-includes --cpp-ptr-type flatbuffers::unique_ptr  --no-fb-import -I include_test monster_test.fbs monsterdata_test.json
+../flatc --python --gen-mutable --reflect-names --gen-object-api --gen-compare --cpp-ptr-type flatbuffers::unique_ptr  --no-fb-import -I include_test monster_test.fbs
```

Comment:
Why does this need a new, separate Python line?

Reply:
Ah nevermind, looks like you are actively working on this PR still :-)

Reply:
Because this PR enables "--no-includes" for Python, meaning with "--no-includes", the import statements will not be generated. This is useful when all generated Python files are concatenated into one single file. However, in the generated example, each table/struct has its own generated file. Therefore, "--no-includes" should not be turned on.

Reply:
@lu-wang-g Hmm should we test both with and without that flag? cc @aardappel

Reply:
As far as I know, the there's a test that executes both generate_code.sh and generate_code.bat, and then compares the generated file with the examples checked in. Therefore, only one version (either with --no-includes or without) can be tested. 

However, the one with --no-includes (meaning the generated code does not have import statements) will break PythonTest.sh. Basically, since unlike other language, in Python each table and struct has their own classes in a separate file, the import statements are indispensable. However, --no-includes is helpful if the generated files are later all concatenated in one big joint file, which is an internal feature at this point.

Reply:
Yes, we're really only set up to test one set of flags. It would be best if that was whatever the user is most likely to be using, but.. whatever works.

Reply:
Sounds good! Thanks Wouter!